### PR TITLE
jamela2 part 1 is added

### PR DESCRIPTION
**Pull request by samim1991**

### CODE REVIEW

**Comment by hudams** [comment]
Joss Code vai

**Comment by juneduzzaman** [comment]
good job, keep up the good work.

**Comment by ruhulanwar** [comment]
fatafati!

File: src/jamela2.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+public class jamela2 {
+
+    int a=2;
+}
```
Comment (hudams) [inline src/jamela2.java:4]: where is println?

**Review by nyanwar1 — APPROVED** [review]

**Review by hudams — APPROVED** [review]
Looks good to me?!

**Review by juneduzzaman — APPROVED** [review]
*****

**Review by ruhulanwar — APPROVED** [review]
awesome!

**Pull request merged**